buildspec.yml

```diff
@@ -18,9 +18,9 @@ phases:
       - echo Building the Docker image...
       - |
         if expr "${CODEBUILD_BUILD_ARN}" : ".*tensorflow-1" >/dev/null; then
-          python src/main.py --buildspec $FRAMEWORK/buildspec-tf1.yml --framework $FRAMEWORK
+          python src/main.py --buildspec $FRAMEWORK/buildspec-tf1-ei.yml --framework $FRAMEWORK
```

Comment:
Buildspec are copied from #327 and might be modified sightly. All the buildspec changes will be removed for the final revision after the merging of #327.